backend/globaleaks/db/migrations/update_31/__init__.py

```diff
@@ -182,6 +189,9 @@ def migrate_Node(self):
                 new_node.contexts_clarification = old_node.context_selector_label
                 continue
 
+            if v.name == 'whistleblowing_receipt_prompt':
+                continue
```

Comment:
@evilaliv3 Is the default value for `node.whistleblowing_receipt_prompt` set in`self.update_model_with_new_templates`?

If not you need to assign it. Otherwise I will merge!